kotlintest-assertions/src/commonMain/kotlin/io/kotlintest/matchers/string/matchers.kt

```diff
@@ -281,3 +281,22 @@ fun beEqualIgnoringCase(other: String) = neverNullMatcher<String> { value ->
     "${value.show()} should not be equal ignoring case ${other.show()}"
   )
 }
+
+fun String?.shouldBeTruthy() = this should beTruthy()
+fun String?.shouldBeFalsy() = this should beFalsy()
+
+fun beTruthy() = object : Matcher<String?> {
+   override fun test(value: String?) = MatcherResult(
```

Comment:
I think we should extract the array to be a single variable. 
This will avoid having to change different points of code

Reply:
Yeah, you are right, I missed that

Reply:
I'd also suggest to use a list instead of an array, so we don't get tricked into a bad toString (might also suggest to add one test of the exception message)